docs/source/routing/operations/subscriptions/index.mdx

```diff
@@ -341,6 +341,22 @@ The router considers subscription operations **identical** if all of the followi
 - The operations sent to the subgraph have identical GraphQL selection sets (i.e., requested fields).
 - The operations provide identical values for all headers that the router sends to the subgraph.
 
+### Configuring deduplication
+
+You can ignore specific headers when doing the subscription deduplication. If you propagate `user-agent` header to subgraphs and you still want to deduplicate subscriptions and so ignore this header
+in the deduplication algorithm you can do it by adding the following to your router's YAML config file:
```

Comment:
```suggestion
You can ignore specific headers when deduplicating subscriptions.

If you propagate a header with varying values to your subgraph (for example, `User-Agent`), but you still want to deduplicate subscriptions and you can accept that only one out of many values will be propagated, you can configure `ignored_headers` like this:
```

I'm not sure this is the best way to phrase it, but I think it'd be good to explicitly state that you lose some detail by doing this. Maybe @apollographql/docs can help or follow up with an improvement PR?